build.rs

```diff
@@ -54,18 +61,33 @@ fn main() {
         .expect("Couldn't write bindings!");
 }
 
-fn build_dtrace() {
-    Command::new("git")
-        .args(&[
-            "clone",
-            "https://github.com/microsoft/DTrace-on-Windows.git",
-            "target\\dtrace",
-        ])
-        .output()
-        .expect("Failed to clone dtrace");
+fn build_dtrace() -> anyhow::Result<()> {
+    let output = if !PathBuf::from(DTRACE_SRC_DIR).is_dir() {
+        Command::new("git")
+            .args(&[
+                "clone",
+                "https://github.com/microsoft/DTrace-on-Windows.git",
+                DTRACE_SRC_DIR,
+            ])
+            .output()
+            .expect("Failed to clone dtrace");
+    } else {
+        Command::new("git")
+            .arg("udpate")
+            .current_dir(DTRACE_SRC_DIR)
+            .output()
+            .expect("Failed to update dtrace");
+    };
 
-    let _ = Command::new("powershell")
-        .args(&[".\\build-dtrace.ps1"])
+    println!("> git update/clone {output:?}");
+
+    let output = Command::new("powershell.exe")
```

Comment:
We don't need this output, build logs get generated inside dtrace directory not in `stdout`